server/protocols/jmap-rfc-8621-integration-tests/jmap-rfc-8621-integration-tests-common/src/main/scala/org/apache/james/jmap/rfc8621/contract/MailboxChangesMethodContract.scala

```diff
@@ -1407,7 +1399,6 @@ trait MailboxChangesMethodContract {
            |        "oldState": "$newState",
            |        "newState": "$newState",
            |        "hasMoreChanges": false,
-           |        "updatedProperties": [],
```

Comment:
I think we got an issue here as `implicit null value filtering` will cause back-references to fail.

Can we find a way to rather have `"updatedProperties": null`?

Can you write 2 tests involving back-references (Mailbox/changes & Mailbox/get) when `"updatedProperties": null` and when `"updatedProperties": ["totalEmails", "unreadEmails", "totalThreads", "unreadThreads"]` ?